### PR TITLE
Better now

### PR DESCRIPTION
**Pull request by loctauxphilippe**
I think yeah

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,14 @@ A custom Dock Extension for Gnome Shell. This Extension is part of the Atom Exte
 ### Compatibility
 
 3.10	| ???
```
Comment (tomvanderlee) [inline README.md]: Why not like so:

    | Gnome | Compatible |
    | :---: | :---: |
    | 3.10 | ??? |
    | 3.12 | yes |
    | 3.14 | yes |
    | 3.16 | ??? |`

Would become:

| Gnome | Compatible |
| :---: | :---: |
| 3.10 | ??? |
| 3.12 | yes |
| 3.14 | yes |
| 3.16 | ??? |

Comment (kyuucr) [inline README.md]: Or more simple:
```
[triple backtics]
3.10 | ???
3.12 | Yes
3.14 | Yes
3.16 | ???
[triple backtics]
```

would become:

```
3.10 | ???
3.12 | Yes
3.14 | Yes
3.16 | ???
```

**Comment by loctauxphilippe** [comment]
@kyuucr @tomvanderlee Do whatever you want guys. I proposed something, you can edit it!

**Comment by satya164** [comment]
Why not just list the compatible Gnome Shell versions separated by comma? A table is overkill.

**Comment by cldx** [comment]
Does it really matter? A table just looks nicer. It says "Hey look, we even added a table for your convenience. That's how much we love you :)"

**Comment by tomvanderlee** [comment]
Fixed in 9644b92 and efd7b7a8cd

**Pull request closed without merging**